service/schema/AttributeTypeState.kt

```diff
@@ -96,21 +97,24 @@ class AttributeTypeState internal constructor(
 
         schemaSrv.mayRunReadTx { tx ->
             val typeTx = conceptType.asRemote(tx)
-            typeTx.getOwnersExplicit(true).forEach {
-                load(it, isKey = true, isInherited = false)
-            }
-            typeTx.getOwnersExplicit(false).filter { !loaded.contains(it) }.forEach {
-                load(it, isKey = false, isInherited = false)
-            }
-            typeTx.getOwners(true).filter { !loaded.contains(it) }.forEach {
-                load(it, isKey = true, isInherited = true)
-            }
-            typeTx.getOwners(false).filter { !loaded.contains(it) }.forEach {
-                load(it, isKey = false, isInherited = true)
+            val typeName = typeTx.label.name()
+            if (!schemaSrv.loadedTypeState.contains(typeName, ownerTypes)) {
```

Comment:
If our loaded state doesn't contain this type of data loaded for the given type name, load it. We instantly append to loadedState rather than at the end to prevent loops.